js/decklist/main.js

```diff
@@ -572,6 +578,10 @@ function statusAndTooltips(valid) {
           notifications.push(prop, ['DCI number must contain only numbers', validType]);
         } else if (validationObject['error'] === 'toolarge') {
           notifications.push(prop, ['DCI numbers must be 10 digits or less', validType]);
+        } else if (validationObject['error'] === 'invalid') {
+          notifications.push(prop, ['DCI number is invalid', validType]);
```

Comment:
If they are getting the invalid issue they entered an invalid 8/10 digit number or a 9 digit number (which is never valid)